UPGRADE-3.0.md

```diff
@@ -0,0 +1,32 @@
+# Upgrading from 2.x to 3.0
+
+API client release 2.0 contains few backward incompatible changes.
+
+This guide will help you upgrade your codebase.
+
+## `UserRecommendation::create()` now accepts only `$user_id` and `$scenario` 
+`UserReccomentation::create()` accepts only two argumens: `$user_id` and `$scenario`.
+Both are arguments are required. Reccomendation command can be further parametrized
+using fluent API.
```

Comment:
Also here could be mentioned that default parameters are now being set using Matej Admin Console and setters only override them.

Reply:
Same as above